Source/WebCore/Modules/modern-media-controls/controls/macos-fullscreen-media-controls.js

```diff
@@ -175,12 +176,16 @@ class MacOSFullscreenMediaControls extends MediaControls
 
         this._lastDragPoint = this._pointForEvent(event);
 
-        this.element.addEventListener("mousemove", this, true);
         this.element.addEventListener("mouseup", this, true);
     }
 
     _handleMousemove(event)
     {
+        if (this._lastDragPoint === undefined) {
```

Comment:
Nothing wrong with such an explicit check, but since distinguishing between `null` and `undefined` is not strictly required here, I would go with a null-ish check and just write:
```suggestion
        if (!this._lastDragPoint) {
```
I think also a comment explaining that this check captures the non-dragging state could prove useful.